packages/trader/src/Modules/Trading/Containers/trade-params-mobile.tsx

```diff
@@ -12,6 +10,62 @@ import React from 'react';
 import classNames from 'classnames';
 import { localize } from '@deriv/translations';
 
+type TTradeParamsModal = {
+    is_open: boolean;
+    toggleModal: () => void;
+};
+
+type TTextValueStrings = {
+    text: string;
+    value: string;
+};
+
+type TTradeParamsMobile = {
+    currency: string;
+    toggleModal: () => void;
+    isVisible: (component_key: string) => boolean;
+    setAmountTabIdx: (amount_tab_idx?: number) => void;
+    amount_tab_idx?: number;
+    setTradeParamTabIdx: (trade_param_tab_idx: number) => void;
+    trade_param_tab_idx: number;
+    setDurationTabIdx: (duration_tab_idx?: number) => void;
+    duration_unit: string;
+    duration_units_list: Array<TTextValueStrings>;
```

Comment:
```suggestion
    duration_units_list: TTextValueStrings[];
```
nit: looks more readable this way?